internal/db/repo.go

```diff
@@ -429,24 +429,26 @@ func (repo *Repository) UpdateSize() error {
 	return nil
 }
 
-// ComposeMetas composes a map of metas for rendering external issue tracker URL.
+// ComposeMetas composes a map of metas for rendering SHA1 URL and external issue tracker URL.
 func (repo *Repository) ComposeMetas() map[string]string {
-	if !repo.EnableExternalTracker {
-		return nil
-	} else if repo.ExternalMetas == nil {
-		repo.ExternalMetas = map[string]string{
-			"format": repo.ExternalTrackerFormat,
-			"user":   repo.MustOwner().Name,
-			"repo":   repo.Name,
-		}
+	if repo.ExternalMetas == nil {
+		repo.ExternalMetas = make(map[string]string)
+	}
```

Comment:
We do not need to re-create metas map every time this method is being called:
```suggestion
	if repo.ExternalMetas != nil {
		return repo.ExternalMetas
	}
	
	repo.ExternalMetas = make(map[string]string)
```